libs/config/src/lib/applications.config.spec.ts

```
@@ -0,0 +1,61 @@
+      process.env['USER_HOST'] = 'user.example.com';
+      process.env['USER_PORT'] = '5555';
+
+      const config = userAppConfig();
+      expect(config).toEqual({
+        protocol: 'https',
+        host: 'user.example.com',
+        port: '5555',
+        name: 'user',
+      });
+
+      delete process.env['PROTOCOL'];
```

Biome

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)